src/components/filters/checkbox.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
   apply: (filtersToApply) => void
   selected?: (string | number)[]
   facet?: { [key: string]: number }
+  tracking?: (value) => void
```

Comment:
Can we add a type to the value here?

Reply:
@Averethel  We can but I was thinking now that this function is not only for tracking maybe is not limited to a type.